src/test/data/JsonSerializableAddressBookTest/typicalStudentsAddressBook.json

```diff
@@ -5,42 +5,49 @@
     "phone" : "94351253",
     "email" : "alice@example.com",
     "address" : "123, Jurong West Ave 6, #08-111",
-    "tags" : [ "friends" ]
+    "tags" : [ "friends" ],
+    "remark" : "Weak at Math"
   }, {
     "name" : "Benson Meier",
     "phone" : "98765432",
     "email" : "johnd@example.com",
     "address" : "311, Clementi Ave 2, #02-25",
-    "tags" : [ "owesMoney", "friends" ]
+    "tags" : [ "owesMoney", "friends" ],
+    "remark" : "Weak at English"
```

Comment:
great example remarks used here

Reply:
Agree, since not every student comes with a remark when creating the instance.